pkg/controller/job/controller_test.go

```diff
@@ -625,7 +629,7 @@ func TestWatchJobs(t *testing.T) {
 	// and make sure it hits the sync method.
 	stopCh := make(chan struct{})
 	defer close(stopCh)
-	go manager.jobController.Run(stopCh)
+	go manager.Run(0, stopCh)
```

Comment:
> 0 for number of workers is very non-intuitive.
> If 0 is expected here, can you please add a comment why?

It was started by the next line.  I just updated to one and removed the old line.